### ISSUE TITLE
P/R/O output nodes 

### ISSUE THREAD

**Issue by waquey** [AITTSMD/MTCNN-Tensorflow#185]
Hello, I'd like to freeze the graph and save them as pb 
Currently I've tried to dump the p-net and set "conv4_1/BiasAdd" and "conv4_2/BiasAdd" as output nodes and it can inference successfully. 

However, I'm not sure if R-net/O-net output nodes are as following? 

bbox_fc1/BiasAdd
landmark_fc1/BiasAdd

Because I have some errors like: 

InvalidArgumentError (see above for traceback): Input to reshape is a tensor with 26496 values, but the requested shape has 221184
         [[Node: import/Flatten/Reshape = Reshape[T=DT_FLOAT, Tshape=DT_INT32, _device="/job:localhost/replica:0/task:0/device:CPU:0"](import/conv3/add, import/Flatten/concat)]]

1. Should I correct the output node names for R/O Net? 
2. mtcnn_model defined R/O's input as 384x24x24x3 and 384x48x48x3, can I just create a new Placeholder as None, 24, 24, 3 while restoring? 

Thanks

**Comment by zm931116** [comment]
yes ,you can, because the size of the first dimession doesn't really matter, the acutal input of RNet depends on the output of PNet.

**Comment by liangheng** [comment]
@waquey hello have you freeze the graph and save three net .ckpt to .pb file ? I have the same need, could you give me some advice

**Comment by 17759205390** [comment]
@waquey have you solve？I need ,too.
